fundamentos/bloco-05-javascript-dom-eventos-e-web-storage/dia-2-javascript-trabalhando-com-elementos/exercicio-fixacao1.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+<main id="paiDoPai">
+    <section id="pai">
+       <section id="primeiroFilho"></section> <!--removido -->
+      <section id="elementoOndeVoceEsta">
+        <section id="primeiroFilhoDoFilho"></section>
+        <section id="segundoEUltimoFilhoDoFilho"></section>
+      </section>
+      Atenção!
+      <section id="terceiroFilho"></section>
+      <section id="quartoEUltimoFilho"></section>
+    </section>
+</main>
+
+  <script>
+    
+    // parte 1
+    document.getElementById("elementoOndeVoceEsta").parentNode.style.color = "pink";
+    
+    document.getElementById("primeiroFilhoDoFilho").innerText = "Hello world"
+
+    console.log(document.getElementById("pai").firstElementChild)
+
+    console.log(document.getElementById("elementoOndeVoceEsta").previousElementSibling);
+
+    console.log(document.getElementById("elementoOndeVoceEsta").nextSibling);
+
+    console.log(document.getElementById("elementoOndeVoceEsta").nextElementSibling);
+
+    console.log(document.getElementById("pai").lastElementChild.previousElementSibling)
+
+    console.log("-----------");
+
+    // parte 2
+
+    let irmao1 = document.createElement("div");
+    
+    document.getElementById("elementoOndeVoceEsta").parentElement.appendChild(irmao1)
+    console.log(document.getElementById("pai").children);
+    
+    let filho2 = document.createElement("div");
+
+    document.getElementById("elementoOndeVoceEsta").appendChild(filho2);
+    console.log(document.getElementById("elementoOndeVoceEsta").children);
+
+    let filho3 = document.createElement("div");
+
+    document.getElementById("primeiroFilhoDoFilho").appendChild(filho3);
+    console.log(document.getElementById("primeiroFilhoDoFilho").children);
+
+    filho3 = document.getElementById("primeiroFilhoDoFilho").firstElementChild;
+
+    console.log(filho3.parentElement.parentElement.nextElementSibling);
+
+    console.log("-----------");
+
+    //parte 3
+
+    let pai = document.getElementById("paiDoPai").firstElementChild;
+
+    pai.removeChild(pai.firstElementChild);
+
+    let elementoOndeVoceEsta = document.getElementById("paiDoPai").firstElementChild.firstElementChild;
+
+    elementoOndeVoceEsta.removeChild(elementoOndeVoceEsta.lastElementChild)
+
+    elementoOndeVoceEsta.removeChild(elementoOndeVoceEsta.lastElementChild)
+
+    let primeiroFilhoDoFilho = document.getElementById("paiDoPai").firstElementChild.firstElementChild.firstElementChild;
+
+    primeiroFilhoDoFilho.removeChild(primeiroFilhoDoFilho.firstElementChild)
+    
+    pai.removeChild(elementoOndeVoceEsta.nextElementSibling);
+    pai.removeChild(elementoOndeVoceEsta.nextElementSibling);
+    pai.removeChild(elementoOndeVoceEsta.nextElementSibling);
+
```

Comment:
nesta parte voce repete 3 vezes o seguinte codigo `pai.removeChild(elementoOndeVoceEsta.nextElementSibling);`, talvez utilizar um for seria interessante.